### PR TITLE
Azure appconfig perf test

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/appconfiguration/azure-data-appconfiguration-perf/pom.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-data-appconfiguration-perf</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-perf;current} -->
```
Comment (mssfang) [inline sdk/appconfiguration/azure-data-appconfiguration-perf/pom.xml:17]: need to be updated to appconfig related

**Review by joshfree — APPROVED** [review]

**Comment by JonathanGiles** [comment]
Let's get this merged @g2vinay

**Comment by JonathanGiles** [comment]
@g2vinay Could I please get a status update on this PR? Should it be finished and merged, or closed? Thanks!

**Comment by mssfang** [comment]
@JonathanGiles  we already have perf tests for AppConfig. @g2vinay Can you close this PR if you have no other concerns

Link to perf tests: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/appconfiguration/azure-data-appconfiguration-perf

**Pull request closed without merging**